CordovaLib/Classes/Public/CDVViewController.m

```diff
@@ -505,9 +505,10 @@ - (UIView*)newCordovaViewWithFrame:(CGRect)bounds
             self.webViewEngine = [[NSClassFromString(defaultWebViewEngineClass) alloc] initWithFrame:bounds];
         }
     } else {
-        self.webViewEngine = [[NSClassFromString(defaultWebViewEngineClass) alloc] initWithFrame:bounds];
+        WKWebViewConfiguration *config = [self respondsToSelector:@selector(configuration)] ? [self configuration] : nil;
+        self.webViewEngine = [[NSClassFromString(defaultWebViewEngineClass) alloc] initWithFrame:bounds configuration:config];
```

Comment:
It looks like you're only applying this in the case when an engine is not specified.

What might be best is to make a private `initWebViewEngine:(NSClass)engineClass` method that takes in the engine class, and checks if it responds to `initWithFrame:configuration:` and calls that, falling back to `initWithFrame:`.

Then we'd call `initWebViewEngine:` on lines 502, 505, and 509 here

Reply:
This is a great suggestion... I didn't imagine custom web engines using this, but of course, that makes sense.  I'll tidy this up.

Reply:
Since the change was more extensive than expected, I raised it in another branch on my fork: https://github.com/msmtamburro/cordova-ios/pull/1/files

If it makes sense, I'll merge it in to this one.